SPECS/rubygem-test-unit/rubygem-test-unit.spec

```diff
@@ -9,7 +9,6 @@ Distribution:   Mariner
 Group:          Development/Languages
 URL:            https://test-unit.github.io/
 Source0:        https://github.com/test-unit/test-unit/archive/refs/tags/%{version}.tar.gz#/%{gem_name}-%{version}.tar.gz
-Patch0:         remove-missing-devdeps.patch
```

Comment:
I love it, I think we also have to remove the patch file itself.

Reply:
I'm still trying to test if the pipelines would build without the patch. I'll remove the file and the signature once that's finalised.